source/api_reference/single_agreement_reference/index.html.md.erb

```diff
@@ -0,0 +1,97 @@
+---
+title: Get information about a single agreement for recurring payments API reference
+last_reviewed_on: 2022-08-20
+review_in: 6 months
```

Comment:
I wonder if we should set these to a 1/2/3 month cadence? I'm guessing a fair bit will change or be refined along with beta partner technical teams working with the sandbox

Reply:
Ah just noticed that webhooks is set to review in a month, I think the team will be considering the tech + docs in that time anyway so these shouldn't be an issue